gridcomps/History/MAPL_StationSamplerMod.F90

```diff
@@ -0,0 +1,461 @@
+!  Input Data:
+!__ 1. station_id_file: static
+!     plain text format: [id,name,lat,lon,elev]
+
+#include "MAPL_Generic.h"
+module StationSamplerMod
+  use ESMF
+  use MAPL_ErrorHandlingMod
+  use LocStreamFactoryMod
+  use pFIO
+  use MAPL_TimeDataMod
+  use MAPL_VerticalDataMod
+  use MAPL_BaseMod
+  use MAPL_CommsMod
+  use MAPL_LocstreamRegridderMod
+  use, intrinsic :: iso_fortran_env, only: REAL32
+  use, intrinsic :: iso_fortran_env, only: REAL64
+  implicit none
+  private
+
+  public :: StationSampler
+  type :: StationSampler
+     type(LocStreamFactory) :: LSF
```

Comment:
```suggestion
  type :: StationSampler
     private
     type(LocStreamFactory) :: LSF
```
Data should be private to the type.

Reply:
Thanks. I have added this private statement.